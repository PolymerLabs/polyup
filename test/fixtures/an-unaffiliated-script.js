// Part of external-script.html
Polymer('in-the-above-script', {});
