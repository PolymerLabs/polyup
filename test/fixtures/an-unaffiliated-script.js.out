// Part of external-script.html
Polymer({ is: 'in-the-above-script' });
